### PR TITLE
[#176297250] Fix api Award Period

### PR DESCRIPTION
**Pull request by pasqualedevita**

### CODE REVIEW

File: models/award.ts [hunk 1]
```diff
@@ -10,144 +10,144 @@ export class Award {
   fiscal_code: FiscalCode;
 
   @ViewColumn({
-    name: "aw_winn_award_period_id_n"
+    name: "id_n"
   })
-  aw_winn_award_period_id: number;
+  award_winner_id: number;
```
Comment (BurnedMarshal) [inline models/award.ts:15]: Can we add even `id_n` that is the id of the `award_winner`? Could be useful as information.

Comment (BurnedMarshal) [reply models/award.ts:15]: Sorry, that field was added with this PR, we can add `aw_winn_award_period_id_n` that is the id of the period.

Comment (pasqualedevita) [reply models/award.ts:15]: @BurnedMarshal is it ok for you?

Comment (BurnedMarshal) [reply models/award.ts:15]: @pasqualedevita  lgtm. What I mean is that we can add even the field `aw_winn_award_period_id_n` to have the `period_id` useful for debugging purpose.

**Review by BurnedMarshal — APPROVED** [review]

File: models/award.ts (hunk 1 above)
Comment (pasqualedevita) [reply models/award.ts:15]: ok sorry, you are right. added award_winner_period_id

**Pull request merged**